addons/point_of_sale/models/pos_order.py

```diff
@@ -482,6 +483,7 @@ def _prepare_invoice_vals(self):
             'move_type': 'out_invoice' if self.amount_total >= 0 else 'out_refund',
             'ref': self.name,
             'partner_id': self.partner_id.id,
+            'partner_shipping_id': addr['delivery'],
```

Comment:
As point_of_sale module doesn't depend on sale module and `partner_shipping_id` field is defined in the latter one, it will crash if sale module is not installed.

Reply:
nice catch, I updated the PR targeting the correct module :D